### PR TITLE
Update Match Media Store for SSR and Legacy Browsers

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. MatchMedia accepts a defaultQuery parameter
1. Match Media checks to see if `window` and `window.matchMedia` are defined
1. If either are undefined then a stub object is created for the query
 - If the stub object alias matches the default query then it is set as a match
1. MatchMedia has a `getMatches` function that returns the media query matches

### CODE REVIEW

**Comment by areida** [comment]
:star:

**Comment by areida** [comment]
:star:

File: store/match-media.js [hunk 1]
```diff
@@ -9,39 +9,63 @@ var Fluxxor = require('fluxxor');
...
+ *
+ * modules.exports = {
+ *     MatchMediaStore : new MatchMediaStore(params)
  * };
  *
  * In a component's getStateFromFlux method:
  *
  * this.setState({
- *     someStateProperty : matchMediaStore.mqls.alias1.matches ? 'this' : 'that'
+ *     mq : matchMediaStore.getMatches()
```
Comment (zpchavez) [inline store/match-media.js]: Then what do you do with `mq`?

**Comment by zpchavez** [comment]
Documentation seems incomplete. I suppose this isn't testable? :interrobang:

**Comment by areida** [comment]
:star:

**Comment by zpchavez** [comment]
:+1:

**Pull request merged**